app.rb

```diff
@@ -1,7 +1,35 @@
 require 'sinatra/base'
+require './lib/player'
+require './lib/game'
+
 class RockPaperScissors < Sinatra::Base
   get '/test' do
-    'test page'
+    "test page"
+  end
+
+  get '/' do
+    erb(:index)
+  end
+
+  post '/name' do
+    @game = Game.create(Player.new(params[:player_name]))
+    redirect '/play'
+  end
+
+  get '/play' do
+    @game = Game.instance
+    erb(:play)
+  end
+
+  post '/move' do
+    @game = Game.instance
+    @game.move(params[:move])
+    redirect '/result'
+  end
+
+  get '/result' do
+    @game = Game.instance
+    erb(:result)
```

Comment:
Really clean and easy to understand. Great use of ".instance" for your Game class.